nf_core/pipeline-template/nextflow.config

```diff
@@ -171,6 +171,13 @@ profiles {
         shifter.enabled        = false
         charliecloud.enabled   = false
     }
+    wave {
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
+        wave.enabled            = true
+        wave.freeze             = true
+        wave.strategy           = 'conda,container'
+    }
```

Comment:
please harshil-align

Reply:
I thought it was? I used @maxulysse's [suggested change](https://github.com/nf-core/tools/pull/2941#discussion_r1584891714) to do so..

Reply:
I need to coin my own whitespace style.. 🤔 

How about we centre-align on `.` and right align anything on the right hand side of `=`, not including quotes? :trollface: 
```suggestion
    wave {
          apptainer.ociAutoPull =             true
        singularity.ociAutoPull =             true
               wave.enabled     =             true
               wave.freeze      =             true
               wave.strategy    = 'conda,container'
    }
```

Reply:
I'd say it's fine within this scope, we don't have to align all equal signs all over.

@ewels why would you do that?

Reply:
all the other equal signs are aligned. consistency! 🙂

Reply:
The `debug` `nextflow.enable.configProcessNamesValidation` isn't 👀 

Fiiine ye who love massive whitespace diffs, I'll do it..

Reply:
Done in e56f5b0d

Reply:
Nice, now only docs are missing